app/models/release.rb

```diff
@@ -35,9 +39,28 @@ def author
   end
 
   def self.find_by_version(version)
-   if version =~ /\Av(\d+)\Z/
-     number = $1.to_i
-     find_by_number(number)
-   end
- end
+    if version =~ /\Av(\d+)\Z/
+      number = $1.to_i
+      find_by_number(number)
+    end
+  end
+
+  private
+
+  def assing_release_number
+    latest_release_number = project.releases.last.try(:number) || 0
```

Comment:
would it be better to an ActiveRecord query that's equivalent to `ORDER BY number DESC LIMIT 1` ?
Not sure if its possible for IDs to be out of sync with release numbers - if not, you could do `project.releases.count + 1`